src/styl/imports/base.styl

```diff
@@ -4,7 +4,7 @@ html, body, :root
 	width 100%
 	height 100%
 	background #fff
-	font-family 'Cabin', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif	
+	font-family 'Cabin', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
```

Comment:
This might cause a merge conflict for #225 - if you can, skip this here and fix it there 😊